playground/compiler-react-18/package.json

```diff
@@ -9,15 +9,15 @@
   },
   "dependencies": {
     "react": "^18.3.1",
-    "react-compiler-runtime": "19.0.0-beta-ebf51a3-20250411",
+    "react-compiler-runtime": "19.1.0-rc.1",
```

Comment:
https://github.com/vitejs/vite-plugin-react/blob/ec1dfb3c453aefc09a2a1d12f9c518460c300b32/.github/renovate.json5#L17-L23
this worked 👍